components/lib/api.js

```diff
@@ -77,6 +90,9 @@ export const registerUser = async (email_address, redirect_to) => {
 
 export const loginUser = async (token) => {
   return await getAPI(apiEndpoints.USER_LOGIN, { k: token })
+    .then((response) => {
```

Comment:
I suspect that since this is going through the `getAPI` call if the user decides to pre-emptively login again (getting a new token), while the old one currently in the browser is expired, they will end up with an error page instead of properly logging in.

Perhaps in the case of `login` we should bypass the `refreshToken` call since we are going to be getting a new token anyways.